public/css/main.css

```diff
@@ -100,6 +114,22 @@ li {
   --color-workspace-item-focus-mg: var(--color-gray-lighten-50);
   --color-workspace-item-focus-bg: var(--color-gray-lighten-55);
 
+  --color-modal-fg: var(--color-main-fg);
+  --color-modal-mg: var(--color-gray-lighten-60);
+  --color-modal-bg: var(--color-gray-lighten-100);
+  --color-modal-inner-border: var(--color-gray-lighten-50);
+  --color-modal-separator: var(--color-gray-lighten-55);
+  --color-modal-shadow: rgba(0, 0, 0, 0.2);
+  --color-modal-overlay: rgba(0, 0, 0, 0.5);
```

Comment:
At some point I need to have a color overhaul such that we can base semi transparent colors off of another color variable directly.